src/doom/p_setup.c

```diff
@@ -1303,7 +1303,7 @@ P_SetupLevel
 }
 
 // [crispy] height of the spawnstate's first sprite in pixels
```

Comment:
This comment doesn't tell the truth anymore. :wink:

Please add another one to the `switch()` block and move this one to the old code below.